examples/bert/npu/user_script.py

```diff
@@ -0,0 +1,19 @@
+from olive.data.component.dataloader import default_dataloader
+from olive.data.registry import Registry
+from olive.platform_sdk.qualcomm.utils.data_loader import FileListCommonDataLoader
+
+
+@Registry.register_dataloader()
+def huggingface_data_to_raw_dataloader(dataset, batch_size, io_config, **kwargs):
```

Comment:
Instead of doing this manually, maybe this is a good time to try doing it automatically in the evaluator like we do in the quantization pass 
https://github.com/microsoft/Olive/blob/main/olive/passes/snpe/quantization.py#L105

Reply:
We have that in SNPE already. Just remove the dataloader under user_scripts.